doc/source/PreConfiguredSites.rst

```diff
@@ -88,31 +96,29 @@ The following is required for building new spack environments and for using spac
    module load python/3.9.2
    module load ecflow/5.8.4
 
-For ``spack-stack-1.6.0`` with Intel, load the following modules after loading miniconda and ecflow:
+For ``spack-stack-1.7.0`` with Intel, load the following modules after loading miniconda and ecflow:
 
 .. code-block:: console
 
-   module use /work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core
+   module use /work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.7.0/envs/ue-intel-centos/install/modulefiles/Core
    module load stack-intel/2022.0.2
    module load stack-intel-oneapi-mpi/2021.5.1
    module load stack-python/3.10.13
-   module available
 
-For ``spack-stack-1.6.0`` with GNU, load the following modules after loading miniconda and ecflow:
+For ``spack-stack-1.7.0`` with GNU, load the following modules after loading miniconda and ecflow:
 
 .. code-block:: console
 
-   module use /work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core
+   module use /work/noaa/epic/role-epic/spack-stack/orion/spack-stack-1.7.0/envs/ue-gcc-centos/install/modulefiles/Core
    module load stack-gcc/10.2.0
    module load stack-openmpi/4.0.4
    module load stack-python/3.10.13
```

Comment:
module load  sp ?

Reply:
NOT in the spack-stack documentation. This goes AFTER module load jedi-fv3-env ... in the jedi-docs